custom_components/rivian/const.py

```diff
@@ -306,16 +298,140 @@
+        entity_description=RivianSensorEntityDescription(
+            name="Offline Mode Enabled",
+            key=f"{DOMAIN}_telematics_privacy_offline_mode_enabled",
+        )
+    ),
+    "thermal/hvac_mobile_status/3rd_row_left_seat_heat_status": RivianSensorEntity(
+        entity_description=RivianSensorEntityDescription(
+            name="3rd Row Left Seat Heat Status",
+            key=f"{DOMAIN}_thermal_hvac_mobile_status_3rd_row_left_seat_heat_status",
+        )
+    ),
+    "thermal/hvac_mobile_status/3rd_row_right_seat_heat_status": RivianSensorEntity(
+        entity_description=RivianSensorEntityDescription(
+            name="3rd Row Right Seat Heat Status",
+            key=f"{DOMAIN}_thermal_hvac_mobile_status_3rd_row_right_seat_heat_status",
+        )
+    ),
+    "thermal/hvac_mobile_status/defrost_defog_status": RivianSensorEntity(
+        entity_description=RivianSensorEntityDescription(
+            name="Defrost/Defog Status",
+            key=f"{DOMAIN}_thermal_hvac_mobile_status_defrost_defog_status",
+        )
+    ),
+    "thermal/hvac_mobile_status/estimated_cabin_temp_mobile": RivianSensorEntity(
+        entity_description=RivianSensorEntityDescription(
+            name="Estimated Cabin Temperature",
+            key=f"{DOMAIN}_thermal_hvac_mobile_status_estimated_cabin_temp_mobile",
+        )
+    ),
+    "thermal/hvac_mobile_status/left_seat_heat_status": RivianSensorEntity(
+        entity_description=RivianSensorEntityDescription(
+            name="Left Seat Heat Status",
+            key=f"{DOMAIN}_thermal_hvac_mobile_status_left_seat_heat_status",
+        )
+    ),
+    "thermal/hvac_mobile_status/left_seat_vent_status": RivianSensorEntity(
+        entity_description=RivianSensorEntityDescription(
+            name="Left Seat Vent Status",
+            key=f"{DOMAIN}_thermal_hvac_mobile_status_left_seat_vent_status",
+        )
+    ),
+    "thermal/hvac_mobile_status/rear_left_seat_heat_status": RivianSensorEntity(
+        entity_description=RivianSensorEntityDescription(
+            name="Rear Left Seat Heat Status",
+            key=f"{DOMAIN}_thermal_hvac_mobile_status_rear_left_seat_heat_status",
+        )
+    ),
+    "thermal/hvac_mobile_status/rear_right_seat_heat_status": RivianSensorEntity(
+        entity_description=RivianSensorEntityDescription(
+            name="Rear Right Seat Heat Status",
+            key=f"{DOMAIN}_thermal_hvac_mobile_status_rear_right_seat_heat_status",
+        )
+    ),
+    "thermal/hvac_mobile_status/right_seat_heat_status": RivianSensorEntity(
+        entity_description=RivianSensorEntityDescription(
+            name="Right Seat Heat Status",
+            key=f"{DOMAIN}_thermal_hvac_mobile_status_right_seat_heat_status",
+        )
+    ),
+    "thermal/hvac_mobile_status/right_seat_vent_status": RivianSensorEntity(
+        entity_description=RivianSensorEntityDescription(
+            name="Right Seat Vent Status",
```

Comment:
Do we want to name these "Front Right Seat..." To be aligned with the naming of "Front Right Door/Window"?

Reply:
I'll change

Reply:
Too bad Rivian themselves didn't stay consistent in the naming of the actual sensor here.